zircon.core/src/commonMain/kotlin/org/hexworks/zircon/api/builder/component/RadioButtonGroupBuilder.kt

```diff
@@ -40,14 +41,16 @@ class RadioButtonGroupBuilder(
         initialIsHidden = isHidden,
         initialTheme = theme,
         initialTileset = tileset
-    )
-
-    override fun createCopy() = RadioButtonGroupBuilder(
-        isDisabled = isDisabled,
-        isHidden = isHidden,
-        theme = theme,
-        tileset = tileset
-    )
+    ).apply {
+        addComponents(*components.toTypedArray())
+    }
+
+    override fun createCopy() = RadioButtonGroupBuilder().apply {
+        isDisabled = this@RadioButtonGroupBuilder.isDisabled
+        isHidden = this@RadioButtonGroupBuilder.isHidden
+        theme = this@RadioButtonGroupBuilder.theme
+        tileset = this@RadioButtonGroupBuilder.tileset
+    }
```

Comment:
same function in GroupBuilder uses 
`    override fun createCopy() = GroupBuilder<T>().also {
        it.isDisabled = isDisabled
        it.isHidden = isHidden
        it.tileset = tileset
        it.theme = theme
    }`
which i think is neater, but either way one should probably match the other

Reply:
Yeah, this makes sense. I'll refactor the ones with the `@`, yours is more readable.